CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.31.0 - 2024/01/16
```

Comment:
We discussed this briefly, but if we're following semantic versioning, it might be time to bump this to `1.0.0` with the included breaking change.

That said, I'm also fine holding off, since we've previously introduced breaking changes without _strictly_ following semantic versioning.

Reply:
Oh right. Yes that's fine by me. We'll wait for @jhollinger's change and I'll switch to 1.0.0.